ironfish/src/network/peers/connections/webRtcConnection.ts

```diff
@@ -127,18 +127,29 @@ export class WebRtcConnection extends Connection {
   /**
    * Encode the message to json and send it to the peer
    */
-  send = (message: LooseMessage): void => {
+  send = (message: LooseMessage): boolean => {
     if (this.shouldLogMessageType(message.type)) {
       this.logger.debug(`${colors.yellow('SEND')} ${this.displayName}: ${message.type}`)
     }
 
     const data = JSON.stringify(message)
-    this.peer.send(data)
+    try {
+      this.peer.send(data)
+    } catch (e) {
+      this.logger.debug(
```

Comment:
Since we're now ignoring all errors, is it possible to lead to bugs we don't know about now?

Reply:
I'll follow up and make a more specific PR to try to catch this specific error